src/encoded/static/components/browse/PublicationSearchView.js

```diff
@@ -0,0 +1,263 @@
+        const {
+            title: origTitle, // We use "title" here, not "display_title" (which contains year+author, also)
+            "@id" : id,
+            abstract: origAbstract = null,
+            authors = null
+        } = result;
+        const charsPerLine = PublicationSearchResultTitle.maxCharsToShowPerLine(colWidth - 45); // -45 re: ToggleBtn width
+        const titleLen = origTitle.length;
+        const titleMaxLen = charsPerLine * 3; // 3 lines max for it
+        let title = origTitle;
+        if (titleLen > titleMaxLen) {
+            title = title.slice(0, titleMaxLen) + "...";
+        }
+        let abstract = null, abstractMaxLen;
+        if (origAbstract && (!authors || authors.length < charsPerLine) && titleLen < charsPerLine){
+            abstractMaxLen = titleMaxLen;
+            abstract = origAbstract;
+            if (abstract.length > abstractMaxLen) {
+                abstract = abstract.slice(0, abstractMaxLen) + "...";
+            }
+        }
+        return (
+            <React.Fragment>
+                <TableRowToggleOpenButton onClick={toggleDetailOpen} open={detailOpen} />
+                <div className="title-inner overflow-hidden">
+                    <h5 className="mt-0 mb-0 text-500">
+                        <a href={id} onClick={this.onClickTrack}>{ title }</a>
+                    </h5>
+                    <AuthorsBlock authors={authors} maxCharacters={titleMaxLen} />
+                    { abstract ? <p className="abstract mb-0 mt-08">{ abstract }</p> : null }
+                </div>
+            </React.Fragment>
+        );
+    }
+}
+
+
+const AuthorsBlock = React.memo(function AuthorsBlock(props){
+    const { maxCharacters: maxCharLen = 20, authors = [] } = props;
+    const initAuthorsLen = authors.length;
+    if (initAuthorsLen === 0) {
+        return (
+            <div className="authors-list">
+                <em>No Authors</em>
+            </div>
+        );
+    }
+    const initAuthors = authors.slice(0);
+    const authorsToKeep = [];
+    let charCount = 0;
+    while (charCount < maxCharLen && initAuthors.length){
+        const author = initAuthors.shift();
+        const authorLen = author.length;
+        charCount += authorLen;
+        if (charCount < maxCharLen) {
+            authorsToKeep.push(author);
+        } else {
+            break;
+        }
+    }
+    const authorsAreTrimmed = authorsToKeep.length < initAuthorsLen;
+    return (
+        <div className="authors-list">
+            {
+                authorsToKeep.map(function(authorStr, i){
+                    return <span className="author-name" key={authorStr}>{ authorStr }</span>;
+                })
+            }
+            { authorsAreTrimmed ? <span className="author-name"><em>et al.</em></span> : null }
+        </div>
+    );
+});
+
+const AbstractBlock = React.memo(function AbstractBlock(props){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['AbstractBlock' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4551745)